src/cdk/_index.scss

```diff
@@ -1,4 +1,5 @@
-@forward './overlay/overlay' show overlay;
+@forward './overlay/overlay' show overlay, $z-index-overlay-container, $z-index-overlay,
+  $z-index-overlay-backdrop, $dark-backdrop-background;
```

Comment:
I decided to expose these variables, because I can see them being useful.

Reply:
Could we rename these, then? Probably lalong the lines
```
$overlay-container-z-index
$overlay-z-index
$overlay-backdrop-z-index
$overlay-backdrop-color
```

Reply:
Sure, but can we leave it for a follow-up PR? Changing them will make the changes non-merge-safe and we had some g3 issues with renamed Sass variables in the PR that reworked the API.

Reply:
Yeah, sgtm